gcloud-java-storage/src/test/java/com/google/gcloud/storage/BlobTest.java

```diff
@@ -159,4 +165,129 @@ public void testSignUrl() throws Exception {
     replay(storage);
     assertEquals(url, blob.signUrl(100));
   }
+
+  @Test
+  public void testGetNone() throws Exception {
+    replay(storage);
+    assertTrue(Blob.get(storage).isEmpty());
+  }
+
+  @Test
+  public void testGetOne() throws Exception {
+    expect(storage.get(BLOB_INFO.bucket(), BLOB_INFO.name())).andReturn(BLOB_INFO);
+    replay(storage);
+    List<Blob> result = Blob.get(storage, BLOB_INFO);
+    assertEquals(1, result.size());
+    assertEquals(BLOB_INFO, result.get(0).info());
+  }
+
+  @Test
+  public void testGetSome() throws Exception {
+    List<BlobInfo> blobInfoList = Arrays.asList(BLOB_INFO_ARRAY);
+    expect(storage.get(BLOB_INFO_ARRAY[0], BLOB_INFO_ARRAY[1],
+        Arrays.copyOfRange(BLOB_INFO_ARRAY, 2, BLOB_INFO_ARRAY.length))).andReturn(blobInfoList);
+    replay(storage);
+    List<Blob> result = Blob.get(storage, BLOB_INFO_ARRAY);
+    assertEquals(blobInfoList.size(), result.size());
+    for (int i = 0; i < blobInfoList.size(); i++) {
+      assertEquals(blobInfoList.get(i), result.get(i).info());
+    }
+  }
+
+  @Test
+  public void testGetSomeNull() throws Exception {
+    List<BlobInfo> blobInfoList = Arrays.asList(BLOB_INFO_ARRAY[0], null, BLOB_INFO_ARRAY[2]);
```

Comment:
Do you think we should support that? null value is not allowed in the singular case and I wonder if we want to allow it
in the vargarg case (and potentially hide user errors...)